File: TAP/Lab6/Lab6pb1/Student.cpp
#include<iostream>
#include"Student.h"
using namespace std;

Student::Student(string CNP, string nume, int nota, string email)
{
	this->CNP = CNP;
	this->nume = nume;
	this->nota = nota;
	this->email = email;
}
Student::Student(Student& other)
{
	this->CNP = other.CNP;
	this->nume = other.nume;
	this->nota = other.nota;
	this->email = other.email;
}
int Student::getNota()
{
	return this->nota;
}
string Student::detalii()
{
	return "Studentul " + this->nume + " are nota " + to_string(this->nota);
}

File: ObjOriPro/Lab4/Student.java
public class Student
{
	private String nume;
	private String cnp;
	private static int nrInstante;
	public Student(String nume,String cnp)
	{
		this.nume=nume;
		this.cnp=cnp;
		nrinstante++;
	}
	public static int getNrInstante()
	{
		return nrInstante;
	}
}